src/IdentityServer/Features/Account/Account/Code.cshtml

```diff
@@ -25,7 +25,7 @@
                     }
 
                     <div class="form-group">
-                        <input class="form-control" type="password" placeholder="Code"
+                        <input class="form-control" type="text" placeholder="Code" style="-webkit-text-security: disc;"
```

Comment:
Before this change, password managers/browsers would suggest updating the saved password after entering the code since it resembled a password.